api/sys_plugins.go

```diff
@@ -129,8 +129,16 @@ type GetPluginResponse struct {
 	SHA256  string   `json:"sha256"`
 }
 
+// GetPlugin retrieves information about the plugin.
 func (c *Sys) GetPlugin(i *GetPluginInput) (*GetPluginResponse, error) {
 	path := fmt.Sprintf("/v1/sys/plugins/catalog/%s/%s", i.Type, i.Name)
+
```

Comment:
Since the plugin input -> path transform is occurring multiple times/location, it may be worth adding a small helper function to keep the logic consistent.